driver-legacy/src/main/com/mongodb/MongoClientOptions.java

```diff
@@ -75,6 +75,8 @@ public class MongoClientOptions {
     private final int maxConnectionIdleTime;
     private final int maxConnectionLifeTime;
     private final int maxConnecting;
+    private final long maintenanceInitialDelayMs;
```

Comment:
None of the other properties have an ms suffix.  Be consistent.

Reply:
Snap, fixed. Really thought I was careful this time around.